packages/yuudachi/migrations/1658336159-reports.js

```diff
@@ -0,0 +1,85 @@
+/** @param {import('postgres').Sql} sql */
+export async function up(sql) {
+	await sql.unsafe(`
+		create function next_report(text) returns integer
+		language plpgsql
+		stable
+		as $$
+		declare next_id integer;
+		begin
+			select max(report_id) into next_id from reports where guild_id = $1;
+			if next_id is null then return 1; end if;
+			return next_id + 1;
+		end;
+		$$;`);
+
+	await sql.unsafe(`
+		create function set_updated_at() returns trigger
+		language plpgsql
+		as $$
+		declare
+		_new record;
+		begin
+			_new := new;
+			_new."updated_at" = now();
+			return _new;
+		end;
+		$$;`);
```

Comment:
Does this need to be created again if it already exists?

Reply:
I couldn't find it on my explorer of the db, but might be something I made wrong, will make some tests

Reply:
You we're right, I'll remove the duplicate

Reply:
Removed in b5f67fe